mlir/lib/Dialect/GPU/Pipelines/GPUToNVVMPipeline.cpp

```diff
@@ -164,14 +163,14 @@ void buildLowerToNVVMPassPipeline(OpPassManager &pm,
 } // namespace
 
 namespace mlir {
-namespace test {
-void registerTestLowerToNVVM() {
-  PassPipelineRegistration<TestLowerToNVVMOptions>(
-      "test-lower-to-nvvm",
+namespace gpu {
+void registerGPUToNVVMPipeline() {
+  PassPipelineRegistration<GPUToNVVMPipelineOptions>(
+      "gpu-lower-to-nvvm",
       "An example of pipeline to lower the main dialects (arith, linalg, "
```

Comment:
This is no longer an example, is it?  Could you maybe describe somewhere that is expected to be included into this pipeline in the future (and what is clearly out of scope)?